src/promises/allNoLeaks.ts

```diff
@@ -0,0 +1,30 @@
+// Inspired by https://stackoverflow.com/questions/64928212/how-to-use-promise-allsettled-with-typescript#69451755
+const isRejected = (result: PromiseSettledResult<unknown>) =>
+	result.status === 'rejected';
+const isFulfilled = <T>(result: PromiseSettledResult<T>) =>
```

Comment:
We don't need the <T> here

Reply:
OK, can you help with the resulting lint errors then?